### PR TITLE
[techDebt]: [DEL-3887]: Move ECSSteadyStateCheckTask & KubernetesSteadyStateCheckTask out of 400-rest

### PR DESCRIPTION
**Pull request by jennyjames21**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32554)

### CODE REVIEW

**Comment by jennyjames21** [comment]
trigger messagecheck

File: 930-delegate-tasks/src/main/java/io/harness/serializer/kryo/DelegateTasksKryoRegistrar.java [hunk 1]
```diff
@@ -361,6 +362,7 @@ public void register(Kryo kryo) {
     kryo.register(ClusterSelectionCriteriaEntry.class, 50008);
     kryo.register(RancherResolveClustersTaskParameters.class, 50004);
     kryo.register(RancherResolveClustersResponse.class, 50005);
-    kryo.register(SpotInstCommandRequest.class, 7220);
```
Comment (johannesHarness) [inline 930-delegate-tasks/src/main/java/io/harness/serializer/kryo/DelegateTasksKryoRegistrar.java:364]: Why is that one removed?

**Comment by jennyjames21** [comment]
Merge issue, good catch

**Review by johannesHarness — APPROVED** [review]
> Reviewed 3 of 5 files at r1, 8 of 8 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32554)* status: :shipit: complete! ✓

**Pull request merged**